osm2streets/src/intersection.rs

```diff
@@ -130,6 +130,56 @@ impl Intersection {
             format!("{} ({})", self.id, osm_ids)
         }
     }
+
+    // TODO Use RoadEdge?
+    // This skips the "interior" piece of any loop roads
+    pub fn get_road_sides_sorted(&self, streets: &StreetNetwork) -> Vec<RoadSideID> {
```

Comment:
This would be quite nice to explore in a smaller unit test, because I've found it confusing